local/network.go

```diff
@@ -81,6 +82,9 @@ type localNetwork struct {
 	// This network's genesis file.
```

Comment:
nit: not the file it's data, so I still think `genesisData` and `upgradeData` are the better names